ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -1249,6 +1249,24 @@ FCU_CHWZTC2X_HT2XZTC2X_DFVSC2X:
   - CHWZTC2X
   - HT2XZTC2X
   - DFVSC2X
+  
+FCU_DFVSMC_CHWBRC_STM_RTM_FDPM_ZTC_SRC:
+  description: "FCU with Status, zone control, schedule run"
+  is_canonical: false
+  implements:
+  - FCU
+  - DFVSMC
+  - CHWBRC
+  - STM
```

Comment:
You have DFVSMC called out, so this is a discharge fan -- not a supply fan.  This should be `DTM`.

Reply:
Ah, ok sir, in my case, the points list for this FCU -- we have both return air and supply air so I think I'll use both STM and RTM, for discharge I think it is the zone so ZTC (sensor + setpoint). I'll readjust this to match more closely what we have thanks for the ideas sir.

Reply:
you need to use discharge air temperature though, because it is discharging into a space -- not being supplied to other equipment. its a subtle difference, but it is important for us. please use DTM and RTM.

Reply:
will also adjust now to what you recommend, thanks sir